packages/libs/src/sdk/api/generated/full/apis/NotificationsApi.ts

```diff
@@ -0,0 +1,86 @@
+/* tslint:disable */
+// @ts-nocheck
+/* eslint-disable */
+/**
+ * API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+import * as runtime from '../runtime';
+import type {
+  NotificationsResponse,
+} from '../models';
+import {
+    NotificationsResponseFromJSON,
+    NotificationsResponseToJSON,
+} from '../models';
+
+export interface GetNotificationsRequest {
+    userId: string;
+    timestamp?: number;
+    groupId?: string;
+    limit?: number;
+    validTypes?: Array<string>;
```

Comment:
I don't suppose there's any way to have the items in the array be an enum or string union?

Reply:
Actually there is!

After exploring this, realized I already wrote yet another flask restx hack in for this - made this part of the default supported things when defining an argument.

See the last few commits